### PR TITLE
Closes #26

### LINKED ISSUE #26: General Issue: STEP 1 ADIS - Merge IS with SUPPIS
### Background Information

Step 1 - ADIS workflow: to merge IS with SUPPIS.

### Definition of Done

To obtain final merged immunogenicity dataset.

### PR DESCRIPTION
**Pull request by federicobaratin**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/main/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/main/articles/unit_test_guidance.html#writing-unit-tests-in-admiral)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples.
- [ ] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [ ] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page.
- [ ] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [ ] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging.
- [ ] Address all merge conflicts and resolve appropriately.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: R/data.R [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#' IS Dataset
```
Comment (yangc71) [inline R/data.R:1]: @bms63 - in admiral core we include the below in the data.R file, is it necessary to request the same for admiralvaccine? 
#' @keywords datasets
#' @family datasets
#' @source

File: R/my_first_fcn.R [hunk 2]
```diff
@@ -19,6 +19,8 @@
 #' @examples
 #' hello_admiral(hw = FALSE)
 hello_admiral <- function(hw = TRUE) {
```
Comment (yangc71) [inline R/my_first_fcn.R:21]: Not sure if we intend to merge this dummy function into devel. In general, please follow the function naming convention and programming strategy as outline in the [programming stragety guide](https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy.html)

Comment (bms63) [reply R/my_first_fcn.R:21]: Thanks Annie.  Going to remove once we get a few functions into this package

File: R/data.R (hunk 1 above)
Comment (bms63) [reply R/data.R:1]: I will add in the keywords and datasets.  They are just creating the data themselves.  Not sure how to document.  But I will make an issue to get the code that made it.

**Pull request merged**